megamek/src/megamek/server/Server.java

```diff
@@ -19997,6 +19996,9 @@ private void resolveHeat() {
                 int rhsRoll = Compute.d6(2);
                 int targetNumber = 2;
                 switch (entity.getConsecutiveRHSUses()) {
```

Comment:
Is it just me or is does this switch statement have an unclosed curly bracket? How the hell does this even compile?

Reply:
I don't know, but I suspect that the problem is it cannot be open in the browser or you need to check it by raw data. I have to download GitHub Desktop to apply it.

Also, it seems the reason why I am keep failed to find this - I am constantly find Report(5000); but there is nothing on the search.

Reply:
It just looks like it does because of Github struggling with the file size